src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1366,6 +1366,31 @@ public static EdmTerm AddTerm(this EdmModel model, string namespaceName, string
             return term;
         }
 
+        /// <summary>

```

Comment:
Bonus:  If we can add extension methods to retrieve the annotations using target string, it could be helpful for the OData Uri parser. 
Supposed that when customer is using OData Uri parser and get a ODataPath, then, he maybe need to retrieve all annotations for this ODataPath and make decisions based on the annotations.